README.md

```diff
@@ -294,6 +294,22 @@ const headers = await fetch(url)
   .then(res => res.headers)
 ```
 
+The same applies for `request` too:
+```js
+// Do
+const response = await request(url)
+  .then(res => {
+    response.body.dump().catch(err => {
+      console.error('Error while dumping the body');
```

Comment:
Is there anything to do here? Will the body be cleared in this case?

Reply:
dump can't throw